gatsby-theme-shopify-manager/src/__tests__/ContextProvider.test.tsx

```diff
@@ -75,7 +75,47 @@ describe('ContextProvider', () => {
     await wait(() =>
       expect(shopifyBuySpy).toHaveBeenCalledWith({
         storefrontAccessToken: Mocks.ACCESS_TOKEN,
-        domain: Mocks.DOMAIN,
+        domain: Mocks.MYSHOPIFY_DOMAIN,
+      }),
+    );
+  });
+
+  it('appends ".myshopify.com" to shopName when customDomain evaluates false', async () => {
```

Comment:
Rather than describing how the code works, let's describe inputs/outputs
```suggestion
  it('appends ".myshopify.com" to shopName when shopName is not a custom domain', async () => {
```